server/src/ClientToDBAPI/ContactedPeople/mainRoute.ts

```diff
@@ -63,7 +63,7 @@ ContactedPeopleRoute.get('/allContacts/:minimalDateToFilter', handleInvestigatio
         .then(result => {
             allContactsLogger.info(validDBResponseLog, Severity.LOW);
             const allContactedPersons = result.data.allContactedPeople.nodes;
-            const convertedContacts = allContactedPersons.map((contact: InteractedContact) => ({...contact, ...contact.involvementReason}));
+            const convertedContacts = allContactedPersons.map((contact: any) => ({...contact, ...contact.involvementReason, ...contact.personByPersonInfo.personContactDetailByPersonInfo}));
```

Comment:
why is the contact of type any?

Reply:
man this part is a mess to begin with: the truth is it's not really InteractedContact Either, this whole section needs to be re-typed, I'll open a bug on this subject and a seperate PR :)

Reply:
👍